presto-memsql/src/test/java/io/prestosql/plugin/memsql/TestingMemSqlServer.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.memsql;
+
+import com.google.common.collect.ImmutableSet;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Set;
+
+import static java.util.Objects.requireNonNull;
+
+public class TestingMemSqlServer
+        extends JdbcDatabaseContainer<TestingMemSqlServer>
+{
+    private static final String MEM_SQL_LICENSE = requireNonNull(System.getProperty("memsql.license"), "memsql.license is not set");
+
+    public static final String IMAGE = "memsql/cluster-in-a-box";
+    public static final String DEFAULT_TAG = "centos-7.1.4-516dfe4088-1.9.6-1.6.1";
```

Comment:
just tag. Or inline in the `IMAGE` as `memsql/cluster-in-a-box:centos....`. We cannot override just tag anyway.